config/more_samples-glyf_colr_1.py

```diff
@@ -378,28 +385,29 @@ def main():
         "psName": "-".join((_FAMILY.replace(" ", ""), _STYLE)),
     }
 
+    access_chars = _accessors()
```

Comment:
you could even do without the _accessors() function, and simply do `access_chars = iter(string.ascii_letters)` here

Reply:
Done, thanks